tests/FSharp.AWS.DynamoDB.Tests/Utils.fs

```diff
@@ -28,6 +30,19 @@ module Utils =
         new AmazonDynamoDBClient(credentials, config) :> IAmazonDynamoDB
 
 
+    let clearAttribute (table: TableContext<'T>) (key: TableKey) (attribute: string) =
+        let keyAttr = table.Template.ToAttributeValues key
+        table.Client.UpdateItemAsync(
+            new UpdateItemRequest(
+                TableName = table.TableName,
+                Key = keyAttr,
+                AttributeUpdates = Dictionary(Map.ofSeq [ attribute, new AttributeValueUpdate(Action = AttributeAction.DELETE) ])
+            )
+        )
+        |> Async.AwaitTask
```

Comment:
could make the test async rather than baking this in?

Reply:
In that case pretty much all the tests should be async. It’s probably a good idea but I’ll do this separately